packages/mc-scripts/config/create-webpack-config-for-development.js

```diff
@@ -308,6 +308,12 @@ module.exports = ({
           },
         ],
       },
+      // Fix for react-intl
+      // https://github.com/formatjs/formatjs/issues/143#issuecomment-518774786
+      {
+        test: /\.mjs$/,
+        type: 'javascript/auto',
+      },
```

Comment:
As a result we will transpile any `mjs` file anywhere in `node_modules` again?

Reply:
Hmm I don't think so. The way I understand it is that we're just telling webpack to treat `mjs` as "normal" JS modules so it'll look for `js` modules.
This is just a guess, I couldn't find any good resource about this stuff...

Reply:
https://webpack.js.org/configuration/module/#ruletype

Reply:
Hm, okay. Just want to make sure we don't change bundling behaviour for all deps cause one is borked.

Reply:
I can test it in the mc-fe repo

Reply:
we could also explicity target react-intl...

Reply:
Seems to work fine.